packages/markdown-docx/test/data/ciceroMark/ciceroMark.js

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const paraAndEmphasisJSON = {
+    '$class':'org.accordproject.commonmark.Document',
+    'xmlns':'http://commonmark.org/xml/1.0',
+    'nodes':[
```

Comment:
Could you create a `.cta` archive for this? I can then check with the word add in too.

Reply:
@algomaster99 
Actually, I generated the JSON using the template mark dingus and not `.cta`. However, I will create a `.cta` file for it and send